lib/AuthenticationRequestParameters.ts

```diff
@@ -52,6 +52,8 @@ namespace Msal {
             str.push("state=" + encodeURIComponent(this.state));
             str.push("nonce=" + encodeURIComponent(this.nonce));
             str.push("client_info=1");
+            str.push("slice=testslice");
```

Comment:
I created Issue #25.
Feel free to add a comment here pointing to [https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/25](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/25) if needed

---
In reply to: [113037638](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/23#discussion_r113037638) [](ancestors = 113037638)